colossalai/nn/_ops/embedding.py

```diff
@@ -32,6 +32,8 @@ def colo_embedding_1Dcol(input_tensor: ColoTensor,
     output = ColoTensor.from_torch_tensor(output_parallel, spec=output_spec)
 
     compute_spec = weight.tensor_spec.compute_spec
+
+    return output
```

Comment:
Why ignore `compute_spec.output_replicate`?

Reply:
I added it back

Reply:
reverted